src/screens/WhotoFollow/index.js

```diff
@@ -199,7 +176,7 @@ const WhotoFollow = () => {
           try {
             const userId = await JwtDecode(res.token).user_id;
             const anonymousUserId = await JwtDecode(res.anonymousToken).user_id;
-            setProfileId({
+            setAuth({
               anonProfileId: anonymousUserId,
               signedProfileId: userId,
               token: res.token,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [179-183]

The `setProfileId` function has been replaced with `setAuth`. Make sure that this change doesn't break any other parts of the application that rely on the old function.
```diff
- setProfileId({
+ setAuth({
    anonProfileId: anonymousUserId,
    signedProfileId: userId,
    token: res.token,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->